sdk/helper/certutil/types.go

```diff
@@ -704,6 +704,45 @@ func (n NotAfterBehavior) String() string {
 	return "unknown"
 }
 
+type notBeforeBound int
+
+const (
+	ForbidNotBeforeBound notBeforeBound = iota
+	PermitNotBeforeBound
+)
+
+var notBeforeBoundNames = map[notBeforeBound]string{
+	ForbidNotBeforeBound: "forbid",
+	PermitNotBeforeBound: "permit",
```

Comment:
I'm surprised there's also not a `not_before_duration` limited version as well for parity with TTL and NotAfter?

Reply:
Ho right! I'll add that one.

Reply:
Added a `not_before_bound=duration` option.